projects/twitter/eval_ppl.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
this seems bad that we need this whole bunch of code for every dataset?

Reply:
the submission is just an example here, we can remove it